analyticConfigurations/computed_metric-aws.tag.json

```diff
@@ -18,6 +18,13 @@
           "fqn": "netuitive.aws.applicationelb.httpsuccesspercent",
           "name": "HTTP Success Rate"
         }
+      }, {
+        "match": "netuitive.aws.applicationelb.httpnonerrorpercent",
+        "properties": {
+          "cml": "100 * ${aws.applicationelb.httpcode_target_2xx_count}.actual / (${aws.applicationelb.httpcode_target_2xx_count}.actual + ${aws.applicationelb.httpcode_target_5xx_count}.actual)",
```

Comment:
Does cml have divide by zero checking built into it, such that you don't need to check that 2xx+5xx != 0?

Reply:
It does. I can't find the PR where @jemassey told me it does.